src/components/MyAccount/CheckoutsTab/CheckoutsTab.tsx

```diff
@@ -29,14 +29,19 @@ const CheckoutsTab = () => {
     "Due back by",
     "Manage checkout",
   ]
-
+  const [checkoutRenewing, setCheckoutRenewing] = useState<string>(null)
```

Comment:
checkoutRenewing seems like it should be a boolean. Is there a name that would make it more evident what this variable is? Not sure if it'll be clearer to rename this or refactor the code to change this to a boolean?